gridftp/server/src/test/cmp_alias_ent_test.c

```diff
@@ -34,7 +34,7 @@ int main()
         { .alias = "hell", .alias_len = 4 },
         { .alias = (char[]){(char)-1, (char)0 }, .alias_len = 1 },
         { .alias = "hell?", .alias_len = 5 },
-        { .alias = "hell[o]", .alias_len = 5 }
```

Comment:
It was in there since originally comitted some 7 years ago. Could be a typo, maybe.

Any idea why this didn't made the test to fail?